src/Stryker.Core/Stryker.Core/Options/Inputs/IgnoreLinqExpressionInput.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Stryker.Core.Exceptions;
+using Stryker.Core.Mutators;
+
+namespace Stryker.Core.Options.Inputs
+{
+    public class IgnoreLinqExpressionInput : Input<IEnumerable<string>>
+    {
+        public override IEnumerable<string> Default => Enumerable.Empty<string>();
+
+        protected override string Description => @"The given linq expression will be excluded for this mutation testrun.
+    This argument takes a json array as value. Example: ['linq.FirstOrDefault', 'linq.First']";
+
+        public IEnumerable<LinqExpression> Validate()
+        {
+            if (SuppliedInput is { } && SuppliedInput.Any(w => w.ToLower().StartsWith("linq.")))
+            {
+                var excludedLinqExpressions = new List<LinqExpression>();
+
+                // Get all LinqExpression
+                var linqExpressions = Enum.GetValues(typeof(LinqExpression))
+                    .Cast<LinqExpression>()
+                    .Where(w => w != LinqExpression.None);
+
+                var linqExpressionsInput = SuppliedInput
+                                        .Select(s => s.Split("."))
+                                        .Where(s => s.Length == 2);
```

Comment:
Why should length be 2?

Reply:
The ideia was to only use "linq.{expression} format,  but with this the user never will know he is passing a wrong format
will refactor this